<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <env name="ROSCONSOLE_FORMAT" value="[${severity}] [${node}] ${message}"/>

  <include file="$(find file_server)/launch/ros_sharp_communication.launch"/>

  <arg name="position_scale" default="0.8"/>
  <arg name="orientation_scale" default="0.8"/>
  <arg name="linear_velocity_scale" default="0.15"/>
  <arg name="angular_velocity_scale" default="0.15"/>
  <arg name="use_shared_controller" default="false"/>
  <node name="motion_controller_teleoperation_node" pkg="marslite_control"
        type="motion_controller_teleoperation_node" output="screen">
    <param name="position_scale" value="$(arg position_scale)"/>
    <param name="orientation_scale" value="$(arg orientation_scale)"/>
    <param name="linear_velocity_scale" value="$(arg linear_velocity_scale)"/>
    <param name="angular_velocity_scale" value="$(arg angular_velocity_scale)"/>
    <param name="use_shared_controller" value="$(arg use_shared_controller)"/>
  </node>

  <node name="gripper_pose_publisher_node" pkg="marslite_control"
        type="gripper_pose_publisher_node" output="screen"/>
</launch>